e2e-tests/commands/insurance/eligibility/complete-up-to-specified-eligibility-section-answers.js

```diff
@@ -0,0 +1,43 @@
+import { completeAndSubmitBuyerCountryForm } from '../../forms';
+
+/**
+ * completeUpToSpecifiedEligibilitySectionAnswers
+ * completes eligibility forms up to the specified section
+ * eg, when 'companyDetails' is passed, it will complete all forms up to and including 'companyDetails'
+ * @param {String} section: the section to complete
+ * @param {String} companyNumber: the company number to use in the companies house search form
+ * @param {Boolean} underThreshold: whether the cover period is under the threshold
+ * @param {Boolean} memberOfAGroup: whether the company is a member of a group
+ * @param {Boolean} partyToConsortium: whether the company is a party to a consortium
+ */
+const completeUpToSpecifiedEligibilitySectionAnswers = ({ section, companyNumber, underThreshold, memberOfAGroup, partyToConsortium }) => {
```

Comment:
Can we rename `underThreshold` to `coverPeriodIsUnderThreshold` and then call it below like this?:

```js
cy.completeCoverPeriodForm({ underThreshold:  coverPeriodIsUnderThreshold })
```